From 5c63373cf9ea6a0c28f193b212b7e3e8300f009d Mon Sep 17 00:00:00 2001
From: Luo Jie <quic_luoj@quicinc.com>
Date: Fri, 28 Mar 2025 15:48:39 +0800
Subject: [PATCH 18/44] dt-bindings: interconnect: Add Qualcomm IPQ5424 NSSNOC
 IDs

Add the NSSNOC master/slave ids for Qualcomm IPQ5424 network
subsystem interfaces, which will be used by the gcc-ipq5424
driver that provides the interconnect services by using the
icc-clk framework.

Change-Id: I67843cbac99fd821892c8ad54941f26c2769e210
Signed-off-by: Luo Jie <quic_luoj@quicinc.com>
---
 include/dt-bindings/interconnect/qcom,ipq5424.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/dt-bindings/interconnect/qcom,ipq5424.h b/include/dt-bindings/interconnect/qcom,ipq5424.h
index a77035611..66cd9a9ec 100644
--- a/include/dt-bindings/interconnect/qcom,ipq5424.h
+++ b/include/dt-bindings/interconnect/qcom,ipq5424.h
@@ -20,5 +20,11 @@
 #define SLAVE_CNOC_PCIE3		15
 #define MASTER_CNOC_USB			16
 #define SLAVE_CNOC_USB			17
+#define MASTER_NSSNOC_NSSCC		18
+#define SLAVE_NSSNOC_NSSCC		19
+#define MASTER_NSSNOC_SNOC_0		20
+#define SLAVE_NSSNOC_SNOC_0		21
+#define MASTER_NSSNOC_SNOC_1		22
+#define SLAVE_NSSNOC_SNOC_1		23
 
 #endif /* INTERCONNECT_QCOM_IPQ5424_H */
-- 
2.34.1

